packages/create-app/src/prompts/promptTemplate.ts

```diff
@@ -19,12 +19,23 @@ import { green } from "../highlight.js";
 import { type Template, TEMPLATES } from "../templates.js";
 
 export async function promptTemplate(
-  parsed: { template?: string },
+  parsed: { template?: string; beta?: boolean },
 ): Promise<Template> {
+  let useBeta = parsed.beta ?? false;
   let template = TEMPLATES.find((t) =>
     t.id === parsed.template || t.id === `template-${parsed.template}`
   );
   if (template == null) {
+    const availableTemplates = TEMPLATES.filter(template =>
+      template.hidden !== true
+      && (useBeta
+        ? template.isBeta === true
+        : (template.isBeta === false || template.isBeta == null))
```

Comment:
can we make this change from last time please https://github.com/palantir/osdk-ts/pull/605#discussion_r1747376410